server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -362,9 +369,19 @@ public static String jobConfig(String jobKey, String cfgName) {
             this.put(CPM_PROVIDER, "artemis");
 
             this.put(ACTIVEMQ_ENABLED, "true");
+
             this.put(ACTIVEMQ_EMBEDDED, "true");
+
+            // TODO: Delete the above configuration and only use EMBEDDED_BROKER going forward.
+            // This is currently blocked by complexity in supporting moving configuration paths,
+            // and should coincide with an update to the Configuration object
+            this.put(ACTIVEMQ_EMBEDDED_BROKER, "true");
+            this.put(ACTIVEMQ_JAAS_PROPERTIES_LOGIN_NAME, "PropertiesLogin");
```

Comment:
Maybe the default should be "InVmLogin" or "GuestLogin" to be more precise since we're not actually using the PropertiesLogin module

Reply:
Ahh, I thought that was the convention for the first config name in the JAAS module, but I was mistaken. I've updated this to use InVMLogin as suggested.